build.js

```diff
@@ -13,19 +13,21 @@ var debug = (process.argv.indexOf('--debug') > -1) || (process.argv.indexOf('-d'
 var prod = (process.argv.indexOf('--prod') > -1);
```

Comment:
Looks like this file can die, nej? @

Reply:
Yeah it's only around because there are still bugs and I'm comparing differences :)